demo/project_demo10-FraudDetectionDeltaLake/run.py

```diff
@@ -0,0 +1,331 @@
+from IPython.display import display
```

Comment:
it would be nice if this was integrated in CI somehow.
I discovered that code in blog posts tends to rot if it is not constantly tested